### PR TITLE
Cb 5

### PR DESCRIPTION
**Pull request by mrabiabrn**

### CODE REVIEW

File: practice-app/editPost/editPost.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+editPostDetails_bp = Blueprint('Edit Post Details', __name__)
+@editPostDetails_bp.route('/api/postDetail/<string:username>/<string:postId>', methods=['POST'])
+def editPost(username, postId):
+    
+    db = mongo.db
+
+    postToBeEdited = db.posts.find_one({'id':postId}, {'_id': False})     # find post with the given id 
+    if not postToBeEdited:  
+        abort(500, "Invalid post id")
```
Comment (CodeBlo) [inline practice-app/editPost/editPost.py:15]:
```suggestion
        abort(404, "Invalid post id")  
```

**Review by erencanuysal — APPROVED** [review]

**Review by kocasariumut — APPROVED** [review]
It works as intended.

**Review by efesabanogluu — APPROVED** [review]

File: practice-app/viewPost/viewPost.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+viewPostDetails_bp = Blueprint('View Post Details', __name__)
+
+@viewPostDetails_bp.route('/api/postDetail/<string:postId>', methods=['GET'])
+def viewPost(postId):
+    
+    db = mongo.db
+    postToBeViewed = db.posts.find_one({'id': postId}, {'_id': False})     # find post with the given id 
+
+    if not postToBeViewed:  
+        abort(500, "Invalid post id")
```
Comment (CodeBlo) [inline practice-app/viewPost/viewPost.py:20]:
```suggestion
        abort(404, "Invalid post id")                                           
```

**Review by onurcanavci — APPROVED** [review]

**Review by CodeBlo — APPROVED** [review]

**Review by asi345 — APPROVED** [review]
It works really fine, I think we can merge this branch.

**Review by hamzaakyildiz — APPROVED** [review]

**Review by ramazanbulut — APPROVED** [review]

**Comment by mrabiabrn** [comment]
Approved by all team members

**Pull request merged**